apiary.apib

```diff
@@ -1518,6 +1518,19 @@ Users can be managed only by a super admin.
         "isSuperAdmin": false
     }
     ```
+## Remove User [DELETE /manage/users/{user_id_or_email}]
+It will completly remove user from everywhere (projects, organizations and maintainers).
```

Comment:
Asi by mělo být nějak napsáno, že je potřeba superadmin, aby to mohl člověk udělat...

Reply:
Aha, super! :)